conf/messages.en

```diff
@@ -621,6 +612,24 @@ declaration.itemType.statisticalValue.error.empty = The statistical value cannot
 declaration.itemType.statisticalValue.error.length = Entered statistical value is too long
 declaration.itemType.statisticalValue.error.wrongFormat = Format of this value is incorrect
 
+declaration.commodityDetails.title = Commodity details
+declaration.commodityDetails.combinedNomenclatureCode.header = What is the commodity code?
+declaration.commodityDetails.combinedNomenclatureCode.header.hint = A commodity code is an eight digit number specifically allocated to goods to classify them when moved outside the EU.  For example, 12345678.
+declaration.commodityDetails.combinedNomenclatureCode.error.empty = Commodity code cannot be empty
+declaration.commodityDetails.combinedNomenclatureCode.error.length = Enter a valid commodity code
+declaration.commodityDetails.combinedNomenclatureCode.error.specialCharacters = Commodity code cannot contain special characters
+declaration.commodityDetails.description.header = Describe the goods
+declaration.commodityDetails.description.header.hint = Try to be as specific as you can. For example, 'Copper pipe 15mm x 3m'
+declaration.commodityDetails.description.error.empty = Description cannot be empty
+declaration.commodityDetails.description.error.length = Description cannot be longer than 280 characters
+declaration.commodityDetails.how.header = How do I find a commodity code?
+declaration.commodityDetails.how.step1 = Go to the <a href='https://www.gov.uk/trade-tariff' target='_blank'>UK Trade Tariff</a>.
```

Comment:
Can we extract link for configuration ?

Reply:
Can you inject config into messages file?